server/libs/storage/src/constants/supported-datetime-template.ts

```diff
@@ -0,0 +1,20 @@
+export const supportedYearTokens = ['y', 'yy'];
+export const supportedMonthTokens = ['M', 'MM', 'MMM', 'MMMM'];
+export const supportedDayTokens = ['d', 'dd'];
+export const supportedHourTokens = ['h', 'hh', 'H', 'HH'];
+export const supportedMinuteTokens = ['m', 'mm'];
+export const supportedSecondTokens = ['s', 'ss'];
+export const supportedPresetTokens = [
```

Comment:
I don't think we ended up using these anywhere. Do you want to tackle that in a different PR? We could (above the input field) have a select with these defaults plus a "custom" option that when selected shows the input field. These would need to be changed to have everything wrapped with {{}}.